src/docs/user/references/Variables.adoc

```diff
@@ -153,12 +153,21 @@ The scheduler server verifies that the object exists in the catalog and fulfills
 Examples:
     ** PA:CATALOG_OBJECT will accept "bucket-example/object-example/1539310165443" and "bucket-example/object-example" but not "bucket-example/object-example/153931016" (invalid revision number) and "bucket-example/" (missing object name).
     ** PA:CATALOG_OBJECT(Script, text/x-python) will only accept a catalog object which is a Python script but not a workflow object or a Groovy script.
-    ** PA:CATALOG_OBJECT(,,basic-example, Native_Task) will accept catalog objects that are in buckets ia-basic-example or basic-example-python and named Native_Task or Native_Task_Python but not NATIVE_TASK.
+    ** PA:CATALOG_OBJECT(,,basic-example, Native_Task) will accept catalog objects that are in buckets ia-basic-example or basic-example-python and named Native_Task or Native_Task_Python but not NATIVE_TASK. +
+
++
 Notice that the default behaviour of bucketName or objectName filters can be modified by including the special character `%` that matches any word.
-Hence, by adding `%` in the beginning (resp. end) of a bucketName or an objectName we mean that the filtered value should end (resp. start) with the specified string.
+Hence, by adding `%` in the beginning (resp. end) of a bucketName or an objectName we mean that the filtered value should end (resp. start) with the specified string. +
 Examples:
+
+[none]
+* {empty}
     ** PA:CATALOG_OBJECT(,,%basic-example) will accept catalog objects that are in buckets ia-basic-example or db-basic-example but not the ones in basic-example-python.
     ** PA:CATALOG_OBJECT(,,,Native_Task%) will accept catalog objects that are named Native_Task or Native_Task_Python but not Simplified_Native_Task.
+
++
+NOTE: To accept an empty value as a catalog object, the variable model should be set to `PA:CATALOG_OBJECT?`.
```

Comment:
it is possible to make a PA:CATALOG_OBJECT object variable as option by using the notation PA:CATALOG_OBJECT?. See <add paragraph rederence>.

Reply:
no I mentioned a reference link to the paragraph where optional variables is explained

Reply:
Ah ok I see now. I will add that.